src/main/resources/init.sh

```diff
@@ -13,26 +13,34 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
-#
-
-ACTION=$1
-SERVICE="@serviceName@"
-SERVICE_CMD="service/bin/$SERVICE"
-PIDFILE="var/run/$SERVICE.pid"
-ARGS="@args@"
-
-# uses SERVICE_HOME when set, else, traverse up two directories respecting symlinks
-SERVICE_HOME=${SERVICE_HOME:-$(cd "$(dirname "$0")/../../" && pwd)}
-cd "$SERVICE_HOME"
-
-source service/bin/config.sh
 
 is_process_active() {
    local PID=$1
    ps $PID > /dev/null;
    echo $?
 }
 
+# Everything in this init.sh script relative to the base directory of an SLSv2 distribution
+pushd "`dirname \"$0\"`/../.." > /dev/null
```

Comment:
It's part of the contract, though we do not depend on the functionality.

I'm not so much arguing to keep it...just flagging it violates the spec. Maybe we should just move forward and violate the spec, though.

Sent from my iPhone

On Jul 6, 2016, at 10:35 AM, Robert Fink <notifications@github.com<mailto:notifications@github.com>> wrote:


In src/main/resources/init.sh<https://urldefense.proofpoint.com/v2/url?u=https-3A__github.com_palantir_gradle-2Djava-2Ddistribution_pull_85-23discussion-5Fr69775733&d=DQMCaQ&c=izlc9mHr637UR4lpLEZLFFS3Vn2UXBrZ4tFb6oOnmz8&r=Vp81aOxWZvuVgVK_wp-VF3pIYG92B19LcCw6XKeYC0U&m=6WdP8vnOErn3VWhDX8X78T4VVL1KVGB6njaZYo-TFGE&s=nzEa5DLW6stV2PAAZbQuMe_7LN_FlL0jQdsXxeq1lg0&e=>:

>
>  is_process_active() {
>     local PID=$1
>     ps $PID > /dev/null;
>     echo $?
>  }
>
> +# Everything in this init.sh script relative to the base directory of an SLSv2 distribution
> +pushd "`dirname \"$0\"`/../.." > /dev/null


Is that part of the SLSv2 contract? Do we rely on that functionality somewhere?

—
You are receiving this because you were assigned.
Reply to this email directly, view it on GitHub<https://urldefense.proofpoint.com/v2/url?u=https-3A__github.com_palantir_gradle-2Djava-2Ddistribution_pull_85_files_c4d43f123e5aa09f0f0ef95acf96f33ccb0ec982-23r69775733&d=DQMCaQ&c=izlc9mHr637UR4lpLEZLFFS3Vn2UXBrZ4tFb6oOnmz8&r=Vp81aOxWZvuVgVK_wp-VF3pIYG92B19LcCw6XKeYC0U&m=6WdP8vnOErn3VWhDX8X78T4VVL1KVGB6njaZYo-TFGE&s=jbOfgn-c90Q5dpliw9-42QdiPhYLRrtXYl3rMin5UoM&e=>, or mute the thread<https://urldefense.proofpoint.com/v2/url?u=https-3A__github.com_notifications_unsubscribe_AAp3Q4I-5F6Ug31KqyUZbDkbG2xo1K1rNBks5qS-2DdzgaJpZM4JFzUs&d=DQMCaQ&c=izlc9mHr637UR4lpLEZLFFS3Vn2UXBrZ4tFb6oOnmz8&r=Vp81aOxWZvuVgVK_wp-VF3pIYG92B19LcCw6XKeYC0U&m=6WdP8vnOErn3VWhDX8X78T4VVL1KVGB6njaZYo-TFGE&s=1E-kf6SbVM6AFp230_nkzPlfgDP8az7ezls4fOtkj7o&e=>.